model/validation.go

```diff
@@ -204,16 +204,367 @@ func (t Tags) Validate() error {
 	return errs
 }
 
+// Validate ensures tags are well-formed, but does not insist that tags exist
+func Validate(t map[string]string) error {
```

Comment:
I originally had the validation call `(t Tags) Validate()` but that validation fails if there are 0 tags.  And 0 tags are allowed!  I wanted the validation of the tags, but not the insistence on >0 tags.  Please advise how you would prefer it.